asciidoc/testing/testing_in_nx/frontend_testing_in_nx.adoc

```diff
@@ -20,7 +21,9 @@ To leverage Ensono Stacks and accelerate your testing process within your Nx mon
 
 === Selecting your test framework
 
-We recommend and currently support Playwright within the Ensono Stacks workflow. With its Microsoft backing, extensive feature set, and fast cross-browser web automation capability, it is our preferred tooling choice.
+We recommend and currently support Playwright within the Ensono Stacks workflow. With its Microsoft backing, extensive feature set, and fast cross-browser web automation capability, it is our preferred tooling choice. 
+
+However, existing users can still find information about the deprecated Cypress plugin link:../../getting_started/cypress/plugin-information.adoc[here].
```

Comment:
Do we need to add in the other references to Cypress back into this file?

Reply:
No as no longer supported, so don't want to mention it too much.  Link if need will suffice for existing users.  

Note: i've updated the md equivalent just now as I noticed the following statement was missing.
However, existing users can still find information about the deprecated Cypress plugin here - tested it works.